esp_encrypted_img/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.2.0
+
+### Enhancements:
+- Added an API to get size of struct pre_enc_bin_header as header size which is useful in case of pre_encrypted_ota to check if complete data received or not when config partial HTTP download is enabled: `esp_encrypted_img_get_header_size`
+
```

Comment:
```suggestion
- Added an API to get the size of pre encrypted binary image header, this could be useful while computing entire decrypted image length: `esp_encrypted_img_get_header_size`

```

Reply:
Lets be generic in terms of the change-log entry.